.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,51 @@
+<!-- This is a comment. You can remove it and other comments while filling out the template -->
+
+# Summary
+
+<!-- 
+Please provide a **short** summary in a few sentences of what the PR does.
+Try to keep it to as few full sentences as possible.
+-->
+
+## Checklist
+
+<!--
+Points that don't don't apply can simply be checked off.
+Feel free to add **N/A** for clarity.
+
+Example:
+
+  - [x] Added tests (for code changes) **N/A**
+-->
+
+ - [ ] Marked as draft PR while still working on PR
+ - [ ] Marked as "ready for review" once not in progress
+ - [ ] Filled in all sections
```

Comment:
Filled in all sections? Sorry, that doesn'taadd any value to the checklist. Only to check a item form a check list.

Reply:
Removed